.simplecov

```diff
@@ -48,34 +48,34 @@ require 'coverage'
 Coverage.start
 
 ## ADD SOME CUSTOM REPORTING AT EXIT
-SimpleCov.at_exit do
-  next if $! and not ($!.kind_of? SystemExit and $!.success?)
-
-  header = "#{'*' * 20} SimpleCov Results #{'*' * 20}"
-  results = SimpleCov.result.format!.join("\n")
-  exit_message = <<-EOF
-
-#{header}
-{{RESULTS}}
-{{FAILURE_MESSAGE}}
-
-#{'*' * header.size}
-  EOF
-  percent = Float(SimpleCov.result.covered_percent)
-  if percent < @minimum_coverage
-    failure_message = <<-EOF
-Spec coverage was not high enough: #{percent.round(2)}% is < #{@minimum_coverage}%
-    EOF
-    exit_message.sub!('{{RESULTS}}', results).sub!('{{FAILURE_MESSAGE}}', failure_message)
-    @output.puts exit_message
-    abort(failure_message) if @generate_report
-  elsif @running_ci
-    exit_message.sub!('{{RESULTS}}', results).sub!('{{FAILURE_MESSAGE}}', <<-EOF)
-Nice job! Spec coverage (#{percent.round(2)}%) is still at or above #{@minimum_coverage}%
-    EOF
-    @output.puts exit_message
-  end
-end
+# SimpleCov.at_exit do
```

Comment:
seems like 

```ruby
if ENV['CI']
  # Since version 1.0, we now require you to run `SimpleCov` in your test/spec helper, and then
  # run the provided `codeclimate-ruby` binary separately to report your results
   # to Code Climate.
   # More information here: https://github.com/codeclimate/ruby-test-reporter/blob/master/README.md
   no format
else
  remove all the fancy text printing and just use the html formatter
end
```